zerver/models.py

```diff
@@ -1434,12 +1434,16 @@ class MutedTopic(models.Model):
     stream = models.ForeignKey(Stream, on_delete=CASCADE)
     recipient = models.ForeignKey(Recipient, on_delete=CASCADE)
     topic_name = models.CharField(max_length=MAX_TOPIC_NAME_LENGTH)
+    date_muted = models.DateTimeField(default=datetime.datetime(2020, 1, 1, 0, 0, 0, 0))
```

Comment:
Do we need to pass timezone info to `datetime.datetime` here?

Reply:
Actually, the 6th argument is for `microsecond`s.
Haven't added the 7th argument [`tzinfo`](https://docs.python.org/3/library/datetime.html#datetime.datetime) for timezone data.
Although the `hour` and smaller arguments are initialized to 0.
This line could be changed to `default=datetime.datetime(2020, 1, 1)`.